src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 200px;
+  height: fit-content;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: 0.3s all;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    height: 134px;
+    width: 160px;
+  }
+
+  &__infos-box {
+    margin-top: 40px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $font-base-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: normal;
+    color: #616070;
+    margin: 0 0 16px;
+  }
+
+  &__review-box {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    color: $font-base-color;
+  }
+
+  &__price-box {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 24px 0 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: normal;
+    color: #616070;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: bold;
+    line-height: 18px;
+    color: $font-base-color;
+  }
+
+  &__btn {
+    box-sizing: border-box;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 166px;
+    height: 40px;
+    line-height: 40px;
+
+    background-color: $btn-color;
+
+    border-radius: 5px;
+
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #fff;
+
```

Comment:
yeah this empty here was not necessary because font-size and color are related blocks